client/analytics/report/categories/index.js

```diff
@@ -55,6 +57,16 @@ export default class CategoriesReport extends Component {
 				: 'category';
 		}
 
+		filters[ 0 ].filters[ 2 ].settings.onClick = () => {
```

Comment:
This will be a bit brittle, can you use `.filter` to find the right filter instead?

Reply:
Thank you for the tip :) I also thought it is a bit brittle. Improved with commit https://github.com/woocommerce/woocommerce-admin/pull/5649/commits/8204aa31cf41f819b56943977a03b9519a345d29